gateway/1.6/modules/ROOT/pages/_partials/task-config-disable-logs.adoc

```diff
@@ -0,0 +1,147 @@
+//tag::config-disable-logs-intro[]
+
+Anypoint Flex Gateway logging provides insights into startup, shutdown, API requests, API responses, policies, and Fluent Bit. Logging allows developers and administrators to identify and address issues or anomalies. However, disabling log output to Anypoint Platform might become necessary.
```

Comment:
```suggestion
Anypoint Flex Gateway logging provides insights into startup, shutdown, API requests, API responses, policies, and Fluent Bit. Logging enables developers and administrators to identify and address issues or anomalies. However, disabling log output to Anypoint Platform might be necessary.
```
Any chance we can say under which circumstances or for which purposes disabling log output is necessary?